avro2py/codegen.py

```diff
@@ -363,10 +393,15 @@ def bridge_namespaces(from_namespace: List[str], to_namespace: List[str], target
         remaining_to_components = [c for c in paired_remaining_to_components if c is not None]
 
         if remaining_from_components:
+            from_namespace_as_str = '.'.join(from_namespace)
+            if from_namespace_as_str in namespaces_with_children:
```

Comment:
Rather than having to do `'.'.join` on all of these, why don't we just _not_ do the `".".join` in https://github.com/hchasestevens/avro2py/pull/15/files#diff-dd17a87ea1ee28b0a8ef80f33739b424462dfd309d0a7eb4d58b619f0ec386f3R361 and have these be `tuple` instead of `list` (so they're hashable)?